code/datums/diseases/lycancoughy.dm

```diff
@@ -34,12 +35,20 @@
 												"<span class='userdanger'>You howl!</span>")
 				affected_mob.AdjustConfused(rand(6, 8))
 			if(prob(3))
-				var/list/puppytype = list(/mob/living/simple_animal/pet/corgi/puppy, /mob/living/simple_animal/pet/pug, /mob/living/simple_animal/pet/fox)
-				var/mob/living/puppypicked = pick(puppytype)
-				affected_mob.visible_message("<span class='danger'>[affected_mob] coughs up [initial(puppypicked.name)]!</span>", \
-													"<span class='userdanger'>You cough up [initial(puppypicked.name)]?!</span>")
+				if(barklimit <= 10)
```

Comment:
This is still roughly 10*number of crew amount of animals on the station; this should probably just be a global variable attached to the disease and the limit increased to something like 20-30 (which is still a LOT).

Reply:
how do i make it global for this in the proper manner not the 'fethas attempting to code' way.